docs/querying/basics.md

```diff
@@ -218,30 +251,31 @@ For example, the following expression returns the value of
 Note that the `@` modifier always needs to follow the selector
 immediately, i.e. the following would be correct:
 
-    sum(http_requests_total{method="GET"} @ 1609746000) // GOOD.
+    sum( http_requests_total{method="GET"} @ 1609746000 ) // GOOD.
```

Comment:
Please do not change the formatting here, I think it reads better without the spaces.

Reply:
I would really like to pull attention out to the new placement of the parenthesis.  I agree that the spaces distract.

What do you think about this:  Could we color the outer sum parenthesis to draw attention to them?